libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -32,7 +33,7 @@ export class BookSearchComponent implements OnInit {
   }
 
   ngOnInit(): void {
-    this.store.select(getAllBooks).subscribe(books => {
+    this.subscription = this.store.select(getAllBooks).subscribe((books) => {
```

Comment:
This can be even better if the `books` property is changed to an observable, then the template can use the `async` pipe to manage updates and subscriptions automatically.

```ts
books$ = this.store.select(getAllBooks);
```

```html
*ngFor="let b of books$ | async"
```

Reply:
updated the above changes